### PR TITLE
Exclude human-co-authored commits in `dim-bots`

### PR DESCRIPTION
**Pull request by FloEdelmann**
Test URL: https://github.com/OpenLightingProject/open-fixture-library/commits/c18c0b9caf5127637c56ff518623a0fd3effb738/.github/workflows/test.yaml

The commit from Jun 26, 2020 is dimmed correctly, as the squash-merged PR was opened by a bot.

The commit from Dec 27, 2019 is dimmed incorrectly IMHO, as the squash-merged PR was opened by a bot, but enhanced by me. So also squash merge commit was co-authored by me. This PR fixes that.

### CODE REVIEW

File: source/features/dim-bots.tsx [hunk 1]
```diff
@@ -4,8 +4,7 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-// eslint-disable-next-line import/prefer-default-export
-export const botSelectors = [
+const botSelectors = [
```
Comment (yakov116) [inline source/features/dim-bots.tsx:7]: This is not used anywhere else?

Comment (FloEdelmann) [reply source/features/dim-bots.tsx:7]: Nope :)

Comment (fregante) [reply source/features/dim-bots.tsx:7]: Since https://github.com/sindresorhus/refined-github/pull/3401

**Comment by fregante** [comment]
I'll add this testing URL too, since it's committed and authored by 2 users, but they're both bots: https://github.com/mozilla/web-ext/commits/master

**Pull request merged**